packages/landing/src/config.ts

```diff
@@ -0,0 +1,26 @@
+export const Config = {
+    get BaseUrl(): string {
+        return process.env.TILE_HOST ?? '';
+    },
+    get ApiKey(): string | undefined {
+        return process.env.API_KEY;
+    },
+    get GoogleAnalytics(): string | undefined {
+        return process.env.GOOGLE_ANALYTICS;
+    },
+};
+
+// Inject google analytics after everything has loaded
+if (Config.GoogleAnalytics != null && typeof window != 'undefined') {
+    window.dataLayer = window.dataLayer || [];
+    function gtag(...args: any[]): void {
+        window.dataLayer.push(args);
+    }
+    gtag('js', new Date());
+    gtag('config', `${Config.GoogleAnalytics}`);
+
+    const script = document.createElement('script');
+    script.setAttribute('async', 'true');
```

Comment:
The standard is to set this to an empty string https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute#Syntax

Reply:
Done,